src/scenes/AcceleratorRouter/ParticipantProjects/EditView/ProjectProfileEdit.tsx

```diff
@@ -139,17 +202,22 @@ const ProjectProfileEdit = () => {
     if (
       isComplete(participantProjectUpdateResponse?.status, initiatedRequests.participantProject) &&
       isComplete(assignContactResponse?.status, initiatedRequests.assignTfContact) &&
+      isComplete(updateInternalUsersResponse?.status, initiatedRequests.updateInternalUsers) &&
```

Comment:
Do we need any of the assign contact stuff anymore now that this is replacing it?

Reply:
Good point, will remove.